client/src/pages/CreateProduct.js

```diff
@@ -60,70 +68,164 @@ const typeOfProduct = [
-              <Grid item xs={4}>
-                <Button variant="outlined" fullWidth size="large">Upload</Button>
+            <div className={classes.rootPaper}>
+              <Paper elevation={3}>
+                <input id="1" type="file" onChange={handlePhotoButton} />
+              </Paper>
+              <Paper elevation={3}>
+                <input id="2" type="file" onChange={handlePhotoButton} />
+              </Paper>
+              <Paper elevation={3}>
+                <input id="3" type="file" onChange={handlePhotoButton} />
+              </Paper>
+            </div>
+            <div className={classes.rootPaper}>
+              <Paper elevation={3}>
+                <input id="4" type="file" onChange={handlePhotoButton} />
+              </Paper>
+              <Paper elevation={3}>
+                <input id="5" type="file" onChange={handlePhotoButton} />
+              </Paper>
+              <Paper elevation={3}>
+                <input id="6" type="file" onChange={handlePhotoButton} />
+              </Paper>
+            </div>
+          </Grid>
+          <Grid item xs={6}>
+            <form className={classes.rootForm}>
+              <TextField id="outlined-basic" label="Title" variant="outlined" fullWidth onChange={e => setTitle(e.target.value)} />
+              <TextField id="outlined-basic" label="Description" variant="outlined" fullWidth rows={4} multiline onChange={e => setDescription(e.target.value)} />
+              <Grid container spacing={1}>
+                <Grid item xs={3}>
+                  <FormControl variant="outlined" fullWidth>
+                    <InputLabel htmlFor="outlined-adornment-amount">Amount</InputLabel>
+                    <OutlinedInput
+                      id="outlined-adornment-amount"
+                      label="Amount"
+                      startAdornment={<InputAdornment position="start">$</InputAdornment>}
+                      onChange={e => setAmount(e.target.value)}
+                    />
+                  </FormControl>
+                </Grid>
+                <Grid item xs={9}>
+                  <TextField
+                    id="outlined-select-currency-native"
+                    select
+                    fullWidth
+                    label="Type of Product"
+                    SelectProps={{
+                      native: true,
+                    }}
+                    variant="outlined"
+                    onChange={e => setProductType(e.target.value)}
+                  >
+                    {typeOfProduct.map((option) => (
+                      <option key={option.value} value={option.value}>
+                        {option.label}
+                      </option>
+                    ))}
+                  </TextField>
+                </Grid>
               </Grid>
-            </Grid>
-        </Container>
+            </form>
+          </Grid>
+        </Grid>
+        <Grid className={classes.marginTop64} container justify="center">
```

Comment:
instead of naming these classnames limited to only marginTop, you can name it more specific to this Grid (div) element as to what it's meant to be doing. 

For example I suggest naming this `classes.uploadContainer` and in `uploadContainer` styling, it can still have marginTop of 64 px